siobrultech_protocols/gem/packets.py

```diff
@@ -76,14 +76,16 @@ def __str__(self) -> str:
             }
         )
 
-    @property
-    def type(self) -> str:
-        return self.packet_format.name
-
     @property
     def num_channels(self) -> int:
+        """The number of channels in the packet given the format.  There may be fewer on the device."""
```

Comment:
Fun fact: if there are, the extra ones will likely have garbage values.

Reply:
Oh, I've noticed :D

I've been thinking about how to tackle that particular problem as well.  With the GEM, we at least know we have no more than 32 channels, and we could limit it, despite having more data.

Reply:
The response for `RQSALL` has the actual count among many other things; that's what I'm planning to use for the config-flow version of the Home Assistant component.